renderer/styles/style.css

```diff
@@ -185,4 +210,138 @@ header.mtf-header {
     opacity: 0.2;
     -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=20)";
     filter: alpha(opacity=20);
+}
+
+.question-image{
+    float:left;
+    min-height:1px;
+    width:15%;
+  }
+  
+  .mtf-question-image{
+      height: 18.5vh;
+      float: left;
+      margin-left: -1%;
+      object-fit: fill;
+  }
+  
+  .qc-question-audio-image{
+    width: 6%;
+    -webkit-filter: invert(100%);
+    margin-top: 2%;
+    display: inline-block;
+    float: right;
+  }
+  .mtf-question-audio {
+    display: inline;
+  }
+  
+  .collapse-ques-text {
+      width:74%;
+      min-height: 17.7vh;
+  overflow: hidden;
+  margin: 0px 3% 0px 2%;
+  text-overflow: ellipsis;
+  -webkit-line-clamp: 2;
+  line-height: 1.8;
+  -webkit-box-orient: vertical;
+  display: -webkit-inline-box;
+  }
+  
+  .qc-expand-ques-text{
+      width:74%;
+
+  margin: 0px 3% 0px 2%;
+  text-overflow: ellipsis;
+  line-height: 1.8;
+  -webkit-box-orient: vertical;
+  display: -webkit-inline-box;
+  }
+
+  .mtf-vert-container {
+    text-align: left;
+    width: 33%;
+    float:left;
+    display: block;
+    margin-top: 2%;
+    margin-bottom:5%;
+    height:60%;
+}
+
+.mtf-vert-option {
+    width:100%;
+    cursor: pointer;
+    display: inline-block;
+    text-align: center;
+    vertical-align: middle;
+    margin-bottom: 5%;
+}
+
+
+.mtf-vert-ques-option {
+    margin: 0 7%;
+    border: 1px solid #c8ccd0;
+    height: 100%;
+}
+
+.mtf-vert-ques-text {
+    line-height: 1.5;
+    padding: 5px;
+    display: table;
+    height: 100%;
+    width: 100%;
+}
+
+.mtf-vert-ques-text-inner {
+    position: relative;
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+    word-break: break-word;
+}
+
+.mtf-vert-ques-text-inner p{
+    height:100%;
+    width:100%;
+}
+
+.mtf-vert-ques-image {
+    position: absolute;
+    right:0;
+    top:0;
+}
+
+.mtf-vert-opt-audio-image{
+    text-align:center;
+    background:white;
+    position: absolute;
+    z-index: 1;
+    width: 4vw;
+    height:4vw;
+    bottom:0;
+    right:0;
+    border-radius:50%
+}
+
+.mtf-vert-opt-audio-image img{
+    width:73%;
+    max-width:100%;
+    max-height:100%;
+} 
+
+.qc-popup-close-button {
+    position: absolute;
+    right: -35px;
+    color: white;
+    bottom: 87%;
+}
+
+img.qc-question-fullimage{
+    width:83%;
+    height:100%;
+}
+
+.popup{
```

Comment:
Dont use .popup class to override the default hegiht to 90%. Use the specific class name to override.